### PR TITLE
Analysis Tools - Implemented test cases for JPSurv with Playwright 

### PR DESCRIPTION
**Pull request by sgugulothu**
Analysis Tools - Implemented test cases for JPSurv with Playwright

### CODE REVIEW

File: src/test/java/AnalysisTools/JPSurv/Features/JPSurvHomePagePlaywright.feature [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    Given User navigates to JPSurv home page
+    Then User uploads file in the homepage
+    Then User selects start year of diagnosis
+    Then User selects end year of diagnosis
+    Then User selects interval of maximum years of diagnosis
+    Then User selects both on the NHL and CML checkbox
+    Then User clicks on the submit button to calculate cohert and model specifications
+    Then User changes cohort from NHL to CML
+    Then User clicks on every presentation of nav item
+    Then Verify user is able to hide the configuration panel
```
Comment (hamidnesar) [inline src/test/java/AnalysisTools/JPSurv/Features/JPSurvHomePagePlaywright.feature:134]: Are all of them just Smoke tests? I did not see any of them tagged as @Regression

File: src/test/java/AnalysisTools/JPSurv/Steps/JPSurvHomePagePlaywrightSteps.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+import appsCommon.PlaywrightUtils.Playwright_Common_Utils;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.options.AriaRole;
+import com.nci.automation.web.CommonUtils;
+import com.nci.automation.web.PlaywrightUtils;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+import static com.nci.automation.web.TestProperties.getJpSurvUrl;
+
```
Comment (hamidnesar) [inline src/test/java/AnalysisTools/JPSurv/Steps/JPSurvHomePagePlaywrightSteps.java:16]: Please remove the empty line 16

File: src/test/java/AnalysisTools/JPSurv/Utils/JPSurvConstants.java [hunk 3]
```diff
@@ -2,4 +2,23 @@
...
+
+    /***
+     * NCI ANALYSIS HOMEPAGE TITLE
+     */
+    public static String NCI_ANALYSIS_HOMEPAGE_TITLE = "NCI Analysis Tools Home Page";
+
+    /***
+     * JOB SUBMITTED TEXT
+     */
+    public static String JOB_SUBMITTED_TEXT = "Your job has been submitted";
```
Comment (hamidnesar) [inline src/test/java/AnalysisTools/JPSurv/Utils/JPSurvConstants.java:23]: I would suggest declaring these constants as Final

File: src/main/java/com/nci/automation/web/TestProperties.java [hunk 4]
```diff
@@ -282,7 +282,7 @@ public class TestProperties {
     /**
      * JP-SURV URLS
      */
-    public static String JP_SURV_TEST_URL = "https://analysistools-qa.cancer.gov/jpsurv/";
+    public static String JP_SURV_TEST_URL = "https://analysistools-dev.cancer.gov/jpsurv/";
```
Comment (hamidnesar) [inline src/main/java/com/nci/automation/web/TestProperties.java:285]: The string says TEST URL but the URL is for Dev tier.

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, 
Please review the comments. Thanks!

**Review by hamidnesar — APPROVED** [review]
Looks good! 
Make sure you pull the latest from master

File: src/main/java/com/nci/automation/web/TestProperties.java [hunk 5]
```diff
@@ -4,7 +4,7 @@ public class TestProperties {
 
     public static String ENV = "test";
     public static String BROWSER = "chrome";
-    public static boolean HEADLESS = true;
+    public static boolean HEADLESS = false;
```
Comment (katm007) [inline src/main/java/com/nci/automation/web/TestProperties.java:7]: please change headless back to true

**Review by alenaminsk — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM

**Review by sarwarahmed1 — APPROVED** [review]

**Pull request merged**